osm2pgsql.lua

```diff
@@ -438,6 +440,25 @@ function osm2pgsql.process_node(object)
     if object.tags.place and object.tags.wikidata == 'Q200667' then
         output_hstore['disputed_by'] = 'PK'
     end
+-- Recast Gaza Strip label as country
+    if object.tags.place and object.tags.wikidata == 'Q39760' then
+        output_hstore['place'] = 'country'
+        output_hstore['disputed_by'] = 'SA;PK;ID'
+    end
+-- Recast West Bank label as country
+    if object.tags.place and object.tags.wikidata == 'Q36678' then
+       output_hstore['place'] = 'country'
+       output_hstore['disputed_by'] = 'SA;PK;ID'
+    end
+-- Recast Western Sahara label as country expect for FR;IN;PS;SA;MA;TR;ID;Pl;NL
+    if object.tags.place and object.tags.wikidata == 'Q6250' then
+        output_hstore['place'] = 'country'
+        output_hstore['disputed_by'] = 'FR;IN;PS;SA;MA;TR;ID;Pl;NL'
+    end
+-- Turn off Israel country label for SA;PK;ID
+    if object.tags.place and object.tags.wikidata == 'Q801' then
+        output_hstore['disputed_by'] = 'SA;PK;ID'
```

Comment:
This would result in no country labels for Israel, West Bank, or Gaza?

Instead do you want to set the Arabic, Urdu, and Indonesian names for Israel to Palestine (localized)?

Reply:
(There may be a better way to do this, like having a Palestine label that has a reverse mask.)

Reply:
I can give that a try

Reply:
Confirmed change works in a mini build